docs/json-rpc-interface.md

```diff
@@ -1,69 +1,1080 @@
+"0x0"
+
+### `eth_notePassword`
+#### Parameters
+1. `String` - ignored
+#### Return format
+`false`
+
+### `eth_pendingTransactions`
+#### Description
+Get transaction queue
+#### Parameters
+None
+#### Return format
+`Array` of `Object`s, see `eth_getTransactionByHash`. `sighash` equals to `hash`
+
+### `eth_subscribe`
+#### Description
+HTTP(S): unsupported, always throws exception
+
+WS(S): subscribe for new events/transactions/blocks/stats
+#### Parameters
+1. `String` subscription type:
+ - "logs";
+ - "newPendingTransactions";
+ - "newHeads";
+ - "skaleStats".
+2. Used only when type="logs": object, format is the same as in `eth_newFilter`
+#### Return format
+"0x"-prefixed hex `String` - subscription id
+
+#### Events format
+Depending on the type of subscription, different data is pushed back to the client when requested event happens.
+1. type="logs":
+```
+{
+"jsonrpc":"2.0",
+"method":"eth_subscription",
+"params":{
+    "result":{
+        "address":"0x"-prefixed hex `String`, 20 bytes,
+        "blockHash":"0x"-prefixed hex `String`, 32 bytes,
+        "blockNumber":"0x"-prefixed hex `String`,
+        "data":"0x"-prefixed hex `String`,
+        "logIndex": integer literal,
+        "topics": array of "0x"-prefixed hex `String`s, 32 bytes each,
+        "transactionHash":"0x"-prefixed hex `String`, 32 bytes,
+        "transactionIndex": integer literal
+    },
+    "subscription": "0x"-prefixed hex `String`, subscription id
+    }
+}
+```
+2. type="newHeads"
+```
+{
+"jsonrpc":"2.0",
+"method":"eth_subscription",
+"params":{
+    "result": see `eth_getBlockByHash`, includeTransactions=false,
+    "subscription": "0x"-prefixed hex `String`, subscription id
+    }
+}
+```
+
+3. type="newPendingTransactions"
+```
+{
+"jsonrpc":"2.0",
+"method":"eth_subscription",
+"params":{
+    "result": "0x"-prefixed hex `String`, 32 bytes - transaction hash,
+    "subscription": "0x"-prefixed hex `String`, subscription id
+    }
+}
+```
+
+### `eth_unsubscribe`
+#### Description
+HTTP(S): unsupported, always throws exception
+
+WS(S): unsubscribe from events/transactions/blocks/stats (see `eth_subscribe`)
+#### Parameters
+1. id: decimal or "0x"-prefixed hexadecimal number `String` OR number literal - previously obtained through `eth_subscribe`
+#### Return format
+None
+
+## `personal_*` Methods
+Not supported
+
+## `db_*` Methods
+Not supported
+
+## `shh_*` Methods
+Not supported
+
+## Debug Methods
+### debug_accountRangeAt
+#### Description
+Not supported, always throws
+
+### debug_traceTransaction
+### debug_storageRangeAt
+#### Description
+Not supported, always throws
+
+### debug_preimage
+#### Description
+Not supported, always throws
+
+### debug_traceBlockByNumber
+### debug_traceBlockByHash
+### debug_traceCall
+
+
+## Non-standard Methods
+### oracle_submitRequest
+### oracle_checkResult
+
+#### Description
+Get future transaction queue
+#### Parameters
+None
+#### Return format
+#### Description
+See `eth_pendingTransactions`
```

Comment:
Retrieve to internal api please

Reply:
it was mistakenly left description from debug_getFutureTransactions that was left here